rollbar/__init__.py

```diff
@@ -536,6 +536,49 @@ def wait(f=None):
         return f()
 
 
+def feature_flag(flag_key, variation=None, user=None):
+    """
+    A context manager interface that sets the tags used to model a feature flag.
+
+    key: the key of the feature flag.
+    variation: (optional) the evaluated feature flag variation.
+    user: (optional) the user being evaluated.
+
+    Example usage:
+
+        with rollbar.featureflag('flag1', variation=True, user='foobar@rollbar.com'):
+            code()
+
+    Tags generated from the above example:
+
+    [
+        {'key': feature_flag.key', 'value': 'flag1'},
+        {'key': feature_flag.data.flag1.order', 'value': 0},
+        {'key': feature_flag.data.flag1.variation', 'value': True},
+        {'key': feature_flag.data.flag1.user, 'value': 'foobar@rollbar.com'}
+    ]
+    """
+    flag_key_tag = _create_tag('feature_flag.key', flag_key)
+
+    # create the feature flag order tag
+    order_key = _feature_flag_data_key(flag_key, 'order')
```

Comment:
I was planning to use this on the backend to find the most recent `feature_flag`. What do you think about that?

Reply:
Actually, I removed it. Because it's a list, I think if we preserve the order of the list, we can still determine when each `feature_flag` call was made like this:

https://github.com/rollbar/mox/pull/7097/commits/3faa90d0dd2c97d9c8c39e1c77f03662a2ee6062#diff-62e08d1f82235250b90d7d1cdeb8571c8d09b25b658c650ed074411cacc64baaR166

Reply:
:+1: